src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -179,7 +179,8 @@ export const View = (props: Props) => {
       const exposureStatus = getExposureStatus(
         exposure,
         props.enabledFeatureFlags.includes("AdditionalRemovalStatuses"),
-        isDataBrokerUnderMaintenance(exposure),
+        // TODO: Waiting for criteria for data brokers under maintenance to be determined
```

Comment:
If we want to comment these code out with TODOs, we should mark it with a ticket number so it's easier to search for in the future.

Perhaps use this ticket num `MNTOR-`